fac29/NisaInvest-TFB-FE
# Add form elements

## Description

1. I have added form elements from Shadcn:
- button
-  input
- label
- textarea
- radio-group
- form

2. I have created a LoginForm component as an example form and put it in Login Page. It can be seen by accessing ``/login``

4. Using Zod for form validation
https://zod.dev/?id=introduction

## Related Issue
closes #10
closes #11 
closes #41

## How Can This Be Tested?
Run `` npm install `` > ``` npm run dev ``` and go to ```/login``` to see the form in action.

To create your own form elements eg. SignupForm, refer to shadcn documentation
https://ui.shadcn.com/docs/components/form

LoginForm is a good strarting point, other form elements can be added easily, including radio buttons, textarea etc.
Any change to form elements needs corresponding changes to validation and schema (in the same file)

## Checklist:
